tasks/components/layout/NewsFeed.tsx

```diff
@@ -31,15 +31,16 @@ export type NewsFeedProps = {
  * The right side of the dashboard page
  */
 export const NewsFeed = ({
-  language,
+  overwriteTranslation,
   localizedNews,
   width
-}: PropsWithLanguage<NewsFeedTranslation, NewsFeedProps>) => {
-  const translation = useTranslation(language, defaultNewsFeedTranslations)
+}: PropsForTranslation<NewsFeedTranslation, NewsFeedProps>) => {
+  const translation = useTranslation(defaultNewsFeedTranslations, overwriteTranslation)
   // The value of how much space a FeatureDisplay needs before the title can be displayed on its left
   // Given in px
   const widthForAppearanceChange = 600
-  const usedLanguage = useLanguage().language
+  let usedLanguage = useLanguage().language
+  usedLanguage = overwriteTranslation?.language ?? usedLanguage
```

Comment:
Quite interesting ... we are always need to do this check if we are calling `useLanguage()` directly

Reply:
We only have to do this if we want to overwrite the language and want to use it. `useLanguage()` always returns a value, but to avoid such statements we could add a overwrite to useLanguage which always returns that value if it is set.

```typescript
useLanguage(overwrite?: Languages) {
  ...
  if(!!overwrite} {
    return overwrite
  }
  ...
}
```